Testing/ORKTest/ORKTest/MainViewController.m

```diff
@@ -2306,11 +2306,11 @@ - (void)optionalFormButtonTapped:(id)sender {
     
     //  Test expected multiple choices
     {
-        ORKAnswerFormat *answer = [ORKAnswerFormat choiceAnswerFormatWithStyle:ORKChoiceAnswerStyleMultipleChoice textChoices:[NSArray arrayWithObjects:@"Cat",@"Dog", @"Rock", nil]];
+        ORKAnswerFormat *answer = [ORKAnswerFormat choiceAnswerFormatWithStyle:ORKChoiceAnswerStyleMultipleChoice textChoices:[NSArray arrayWithObjects:@"Cat", @"Dog", @"Rock", nil]];
         ORKQuestionStep *step = [ORKQuestionStep questionStepWithIdentifier:@"question_03"
-                                                                      title:@"Choose the animals?"
+                                                                      title:@"Choose the animals"
```

Comment:
Sorry, I edited my comment and I think you missed it. I think `Select all the animals` is slightly clearer.